src/editor/lib/cameras.js

```diff
@@ -36,7 +36,7 @@ export function initCameras(inspector) {
 
   // Create Inspector camera.
   const perspectiveCamera = (inspector.camera = new THREE.PerspectiveCamera());
-  perspectiveCamera.far = 20000; // Changed from 10000 to 20000
+  perspectiveCamera.far = 100000000; // Changed from 10000 to 20000
```

Comment:
Update the comment as well, just keep `// Changed from 10000` so you won't need to update the comment next time.

Reply:
good idea thanks